src/components/status/PhaseStatusBar.vue

```diff
@@ -48,7 +48,8 @@ export default {
     filters: {
         readable_date(timestamp) {
             // don't forget to convert back to ms for js date handling
-            return moment(timestamp * 1000).format("MM/DD HH:mm:ss");
+            let readable = moment(timestamp * 1000).format("MM/DD HH:mm:ss");
+            return readable
         }
```

Comment:
Can I ask what the purpose of this change is? It looks like it does the same thing - is this just a matter of convention?

Reply:
Oh, I made that change because I was trying out some extra logic in between but ended up removing it and forgot about the original format. You're right, no functional difference.